docs/introduction/role-based-access-control.md

```diff
@@ -91,13 +91,13 @@ The system automatically manages role inheritance:
 
 ```
 ROLE_SUPER_ADMIN (highest)
-    └── ROLE_ADMIN
-        └── ROLE_ALL (all FULL permissions)
-            ├── ROLE_PRODUCT_FULL
-            │   ├── ROLE_PRODUCT_EDIT (includes VIEW)
-            │   ├── ROLE_PRODUCT_CREATE (includes VIEW)
-            │   └── ROLE_PRODUCT_DELETE (includes VIEW)
-            └── [other role hierarchies...]
+    └── ROLE_ALL (all FULL permissions)
+        ├── ROLE_PRODUCT_FULL
+        │   ├── ROLE_PRODUCT_EDIT (includes VIEW)
+        │   ├── ROLE_PRODUCT_CREATE (includes VIEW)
+        │   └── ROLE_PRODUCT_DELETE (includes VIEW)
+        └── [other role hierarchies...]
+ROLE_ADMIN (lowest - login only)
```

Comment:
it is also necessary for `efconnect` and `elfinder` routes, and required for some other basic actions, see `SystemRole::ADMIN` usages. Maybe we could mention it here :thinking: Also, we could also mention, that each administrator has at least this role, right?